### DISCUSSION TITLE
Config creating problem

### DISCUSSION THREAD

**Discussion by alex7070m** [rectorphp/rector#8518] [Q&A] [answered]
![2024-03-02](https://github.com/rectorphp/rector/assets/69157204/bcf0e910-6307-4161-a0a5-bf2bf1704add)
Hello! I'm trying to create config with the vendor/bin/rector command as indicated on the website https://getrector.com/documentation. As a result, an error message appears (in the screenshot). 

PS C:\Users\alexa> vendor/bin/rector

 No "rector.php" config found. Should we generate it for you? [yes]:
 > yes

In RecursiveDirectoryIterator.php line 108:

  RecursiveDirectoryIterator::__construct(C:\Users\alexa\Local Settings\2010-2023_Viber_Media_S.a): Отказано в доступ
   (code: 5)

In RecursiveDirectoryIterator.php line 53:

  RecursiveDirectoryIterator::__construct(C:\Users\alexa\Local Settings\2010-2023_Viber_Media_S.a): Отказано в доступ
   (code: 5)

process [-n|--dry-run] [-a|--autoload-file AUTOLOAD-FILE] [--no-progress-bar] [--no-diffs] [--output-format OUTPUT-FORMAT] [--debug] [--memory-limit MEMORY-LIMIT] [--clear-cache] [--port PORT] [--identifier IDENTIFIER] [--xdebug] [--] [<source>...]

How can this be fixed?

Thanks

**Comment by samsonasik** [comment, marked as answer]
I think that because you're trying install rector directly in "C:\Users\alexa" which a home directory of user "alexa" which has system directory in it which php not allowed to access, try create a project instead, and install rector there, eg:

```
cd C:\Users\alexa
mkdir www
cd www
mkdir someproject
cd someproject
```

Then, setup rector per-project there, ensure that rector config in root of the project

**Comment by alex7070m** [comment]
Thanks for the help! It worked.
